File: Patterns in Java/Pattern_8.java
import java.util.Scanner;

public class Pattern_8 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter No of Rows: ");
    int rows = sc.nextInt();
    
    for (int i = rows; i >= 1; i--) {
      for (int space = 0; space < rows - i; space++) {
        System.out.print("  ");
      }
      for (int j = i; j <= 2 * i - 1; j++) {
        System.out.print(j + " ");
      }
      for (int j = 1; j < i; j++) {
        System.out.print(j + " ");
      }
      System.out.println();
    }
  }
}

File: Patterns in C/Pattern 8.cpp
# include <stdio.h>
int main()
{
	int rows,space;
	printf("Enter No of Rows ");
	scanf("%d",&rows);
	
	for(int i=rows; i>=1; i--)
	{
		for(space=0; space<rows-i; space++)
		{
			printf("  ");
		}
		for(int j=i; j<=2*i-1; j++)
		{
			printf("%d ",j);
		}
		for(int j=1; j<i; j++)
		{
			printf("%d ",j);
		}
		printf("\n");
	} 
	return 0;
}
